_data/navigation_storage.yml

```diff
@@ -0,0 +1,18 @@
+- title: Introduction
```

Comment:
Might it be clearer to title this "Introduction to storage with Gaia" since it's not at first glance very obvious what category / type of the docs the user is viewing otherwise?

Reply:
ok.